doctr/models/detection/core.py

```diff
@@ -82,6 +83,108 @@ def bitmap_to_boxes(
     ) -> np.ndarray:
         raise NotImplementedError
 
+    @staticmethod
+    def nms(boxes: np.ndarray, thresh: float = .5) -> List[int]:
```

Comment:
I would argue that nms should be put in `doctr.utils.metrics` since this will be reused in many architectures and it's a common operation to perform :+1: